styles.css

```diff
@@ -10,6 +10,13 @@
     margin-left: 60px;
 }
 
+.modal.show .modal-dialog {
```

Comment:
I think you wanted to prepend `.fullscreenmodal` here. `.block_qrcode .fullscreenmodal .modal.show .modal-dialog` would be nice, in case anyone else wants to use `fullscreenmodal`.

Reply:
absolutely